docker/dvc_operator.Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.6.7-slim
+
+ENV PYTHONUNBUFFERED 1
+
+RUN apt-get update --fix-missing --no-install-recommends && \
+    apt-get upgrade -y && \
+    apt-get install -y build-essential \
+    python-dev git \
+    libffi-dev libssl-dev libpq-dev \
+    wget && \
+    pip install -U --no-cache-dir pip setuptools wheel dvc && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* && \
+    rm -rf /tmp/*
+
+COPY run_poodle.sh /run_poodle.sh
+
+RUN mkdir /src && cd /src && \
+	git clone https://github.com/yuxiangl/poodle-dvc.git && \
+	cd /src/poodle-dvc && \
+	git checkout dvc-operator && \
```

Comment:
I **love** the idea of making sure we're on the right branch here In prod we want to parametrize it either a) by `pip install`ing the right version of the `bidding` package, or b) checking out the proper branch.